form.yml.erb

```diff
@@ -1,38 +1,34 @@
 <%-
     paths = []
-    paths << Pathname.new("/home/#{User.new.name}")
-    paths << Pathname.new("/data/#{User.new.name}")
     paths << Pathname.new("/scratch/#{User.new.name}")
 
-    projects = User.new.groups.map(&:name).grep(/^pg-./)
+    projects = User.new.groups.map(&:name).grep(/^hb-./)
 
-    paths.concat projects.map { |p| Pathname.new("/data/#{p}")  }
+    paths.concat projects.map { |p| Pathname.new("/scratch/#{p}")  }
 -%>
 
 ---
-cluster: "peregrine"
+cluster: "login1"
```

Comment:
No idea how heavy this is, but would it be worth it using `interactive1` or `interactive2` instead?